.codecov.yaml

```diff
@@ -0,0 +1,15 @@
+github_checks:
+  annotations: false
+coverage:
+  status:
+    project:
+      default: false
+      cli:
+        paths:
+          - "cmd/"
+        target: 75%
+        informational: true
+      pkg:
+        paths:
+          - "pkg/"
+        target: 75%
```

Comment:
So, here we enforce 75% coverage on pkg to merge, right? 
Aren't we well below 75% right now ? Do we need an "informational: true" here too?

Reply:
to be honest I was trying to trigger codecov to post in the PR and I didnt knew how. So I copied the codecov config from elemental-cli directly, but it still wont post the results, no idea why. 

So yeah dont take this into account, its meant to be dropped.

codecov just hates me :/

Reply:
I set it to informational only, just to have the config in there as I cant make codecov work for this PR for some reason :/